boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -417,8 +418,12 @@ unsafe fn setup() -> (
     kernel::debug::assign_gpios(Some(&peripherals.gpio_port[26]), None, None);
 
     // Create a shared UART channel for the console and for kernel debug.
-    let uart_mux = components::console::UartMuxComponent::new(&peripherals.uart0, 115200)
-        .finalize(components::uart_mux_component_static!());
+    // PANIC: 115200 != 0
```

Comment:
It's good that you document that this cannot lead to a panic, but I think the format of this comment intuitively and superficially coveys the exact opposite to me. Maybe be a bit more verbose? "Cannot panic, as 115_200 != 0". Also, it'd be great if you could move the comment to be on the line directly above the unwrap this pertains to.

Same for all the other occurrences.

Reply:
> It's good that you document that this cannot lead to a panic, but I think the format of this comment intuitively and superficially coveys the exact opposite to me.

I use "PANIC" comments describing why a function/method cannot panic in a particular call. In theory, there should be no function/method/closure that certainly panics, so there is no need for a "NO PANIC" comment.

> Also, it'd be great if you could move the comment to be on the line directly above the unwrap this pertains to.

Do you mean putting it between `&peripherals.uart0` and `NonZeroU32::new(115200).unwrap()`?

Reply:
> I use "PANIC" comments describing why a function/method cannot panic in a particular call.

Well, I think that at least to me, that's at least slightly confusing.

> In theory, there should be no function/method/closure that certainly panics, so there is no need for a "NO PANIC" comment.

Well, `panic!()` or `unimplemented!()` exists and is certainly very useful for unreachable code paths. :sweat_smile: 

I realize this is slightly pedantic and I don't feel super strongly about this. I think it could just be clearer if we state that a call to a method that _usually_ can panic, like `.unwrap()`, _will not_ panic for a particular invocation, and state it like that in the comment. Something like "This does not panic, because [...]".

Reply:
I thought using `PANIC` as a comment prefix was a good idea. Similar to `SAFETY` for unsafe blocks. `This does not panic, because [...]` is a bit too verbose for me. Also, searching for `PANIC` with tools such as `grep` is more reliable than an entire phrase which may be split over two lines depending on the initial indentation.

Reply:
Yes, this looks good. Thanks!